File: backend/Plant_tracker/src/main/java/com/example/Plant_tracker/models/Event.java
package com.example.Plant_tracker.models;

// import jakarta.persistence.*;
import java.time.LocalDateTime;
import lombok.Getter;
import lombok.Setter;
import lombok.AllArgsConstructor;


@Getter
@Setter
@AllArgsConstructor
public class Event {
    
    private int idEvent;
    private String type;
    private LocalDateTime date;

}

File: frontend/app/src/main/java/com/example/planttrackerapp/backend/domain/Event.kt
//package com.example.planttrackerapp.domain;
//
//
//import androidx.room.*
//import java.time.LocalDateTime
//
//@Entity(
//    tableName = "events",
//    foreignKeys = [
//        ForeignKey(
//            entity = UserPlant::class,
//            parentColumns = ["id"],
//            childColumns = ["plant_id"],
//            onDelete = ForeignKey.CASCADE
//        )
//    ],
//    indices = [Index(value = ["plant_id"])]
//)
//data class Event(
//    @PrimaryKey(autoGenerate = true) val idEvent: Long = 0,
//    val type: String,
//    val date: LocalDateTime,
//    @ColumnInfo(name = "plant_id") val plantId: Long
//)
